extensions/dash/package.json

```
@@ -15,11 +15,19 @@
       "title": "Find in Dash",
       "subtitle": "Dash",
       "description": "Filter a Dash docset and search for the provided string in it.",
-      "mode": "view"
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "searchstring",
+          "placeholder": "Searchstring",
```

```suggestion
          "placeholder": "Search query",
```
Breaking this up into two words would serve as a more helpful prompt. I think "query" would be more consistent with the Dash convention (as used in the "Web Search" preference pane), plus you can search in Dash for non-string terms like numbers of symbols so this would more accurately convey that.